helm/values.yaml

```diff
@@ -0,0 +1,82 @@
+# Default values for helm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/project-zot/zot
+  pullPolicy: IfNotPresent
```

Comment:
Note: on GCS currently there is no pullPolicy defined in k8s files which defaults to Never (expects image to already be there by previously loading it)